packages/components/data-table-manager/src/data-table-manager.tsx

```diff
@@ -84,7 +84,7 @@ const DataTableManager = <Row extends TRow = TRow>(
         selectedColumns={selectedColumns ?? []}
         customColumnManager={customColumnManager ?? undefined}
       />
-      {props.children
+      {props.children && isValidElement(props.children)
```

Comment:
Do we actually need to use the `isValidElement` function?
Since the `children` prop is already types as a `ReactElement` it seems we shouldn't need to.

Reply:
fixed in [79e91d7](https://github.com/commercetools/ui-kit/pull/3034/commits/79e91d72e7be9ce8dc00a911b81e8a67f566ba0b)